packages/trpc/server/routers/viewer/slots/util.ts

```diff
@@ -480,10 +492,15 @@ export async function getAvailableSlots(input: TGetScheduleInputSchema) {
   };
 }
 
-async function getUserIdFromUsername(username: string) {
+async function getUserIdFromUsername(
+  username: string,
+  organizationDetails: { currentOrgDomain: string | null; isValidOrgDomain: boolean }
+) {
+  const { currentOrgDomain, isValidOrgDomain } = organizationDetails;
   const user = await prisma.user.findFirst({
     where: {
       username,
+      organization: isValidOrgDomain && currentOrgDomain ? getSlugOrRequestedSlug(currentOrgDomain) : null,
```

Comment:
This was required if an org user have the same username with another one without org.